src/jobs/java_isopod_deploy.yml

```diff
@@ -0,0 +1,69 @@
+description: >
+  Deploy java app image to environment using isopod.
+  Supports app-module from monorepo as well as single app from single-app-repo.
+
```

Comment:
For deploying apps to the platform I do not see reason to have stack specific jobs/commands unless they are in separate orbs. Reason is that isopod do not care about stacks..

Reply:
isopod not caring about stacks i agree

me having to use a job that is build to support node+java+go+brainfuck means i always have to fight through a billion optional customizations which i dont need (want) to know

the idea here is to have a job which requires minimal parameters for my java app config, sacrificing reusability for everything except my standard java app. even with that im still ending up with 300 lines that i have to copy+paste to each repo, which is mind-boggling!

Reply:
i understand your comment @michaelkuechler but also agree with @robert-nemet . i propose we adapt the current `deploy_job` and add there are parameter to define where the isopod config file can be found. basically what you did here with `app_path`